### PR TITLE
Business Process Model Execution on Hyperledger Fabric

### PR DESCRIPTION
**Pull request by TimKam**
@Matwog will do the heavy lifting here :-)

### CODE REVIEW

File: _posts/2019-09-18-caterpillar-hyperledger.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+Great.  You can exit out of the cli container and return to your terminal.
+```
+exit
+```
+
+You are now ready to setup Fab3.
+
+
+### Set up Fab3
```
Comment (steffektif) [inline _posts/2019-09-18-caterpillar-hyperledger.md]: Not sure, but in the sentence before you spelled `setup`, now set up. I found this confusing, but English is not my first language

File: _posts/2019-09-18-caterpillar-hyperledger.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+- [Node](https://nodejs.org/en/) (version 8.9.x or greater)
+- [npm](https://www.npmjs.com/) (version 5.6.0)
+
+### Setting up the Hyperledger network
+
+In this step, we will set up fabric locally using docker containers, install the EVM chain code (EVMCC) and instantiate the EVMCC on our fabric peers. This step uses the Hyperledger [fabric-sample](https://github.com/hyperledger/fabric-samples) repo to deploy fabric locally and the [fabric-chaincode-evm](https://github.com/hyperledger/fabric-chaincode-evm) repo for the EVMCC, Fab3 and Fab3.  This step follows the [fabric-chaincode-evm tutorial](https://github.com/hyperledger/fabric-chaincode-evm/blob/master/examples/EVM_Smart_Contracts.md) closely.
+
+### Clean docker and set GOPATH
+
+This will remove all your docker containers and images!
```
Comment (steffektif) [inline _posts/2019-09-18-caterpillar-hyperledger.md]: Might be bold, for more attention, removing stuff from peoples computers makes them angry.

File: _posts/2019-09-18-caterpillar-hyperledger.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+### Method 2: Using the BPMN-Sol compiler
+
+Clone the repository [BPMN-Sol compiler](https://github.com/signavio/BPMN-Sol) to your machine and `cd` into the project directory.
+Run `npm install` to install the dependencies.
+In the `index.ts` file you find a sample XML object that can be compiled to a smart contract.
+Alternatively, you can add your own BPMN file.
+Note that the JavaScript object needs to contain the keys ``bpmn`` with the BPMN XML file as its value and ``name`` with the name of the smart contract as its value.
+
+```
+const xml = {
```
Comment (steffektif) [inline _posts/2019-09-18-caterpillar-hyperledger.md]: I'm glad this doesn't look too bad in the rendered version of markdown, however you might want to put it to a) an extra file or b) a gist?

**Comment by TimKam** [comment]
@steffektif Are you ready to give your approving 👍 ?

**Comment by steffektif** [comment]
I was. 👍

**Pull request merged**